<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>soundSurround</class>
 <widget class="QDialog" name="soundSurround">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="switchSoundSurroundButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{border:0px; color:white;background:transparent;}
QPushButton:hover{background-color:red;}
QPushButton:checked{background-color:red;}</string>
   </property>
   <property name="text">
    <string>开启/关闭7.1环绕</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>591</width>
     <height>471</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QSlider" name="horizontalSliderLeft">
   <property name="geometry">
    <rect>
     <x>833</x>
     <y>130</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {         background: red;         position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */         top: 8px; bottom: 8px;    border-radius: 9px; }     QSlider::handle:horizontal {         width:15px;        border:1px;  background: #008792;         margin: -4px 0; /* expand outside the groove */    border-radius: 15px; }     QSlider::add-page:horizontal {         background: darkgray;     }     QSlider::sub-page:horizontal {         background:lightblue ;     }</string>
   </property>
   <property name="minimum">
    <number>-13</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>-4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>130</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderRight">
   <property name="geometry">
    <rect>
     <x>833</x>
     <y>160</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {         background: red;         position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */         top: 8px; bottom: 8px;    border-radius: 9px; }     QSlider::handle:horizontal {         width:15px;        border:1px;  background: #008792;         margin: -4px 0; /* expand outside the groove */    border-radius: 15px; }     QSlider::add-page:horizontal {         background: darkgray;     }     QSlider::sub-page:horizontal {         background:lightblue ;     }</string>
   </property>
   <property name="minimum">
    <number>-13</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>-4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>160</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderLS">
   <property name="geometry">
    <rect>
     <x>833</x>
     <y>210</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {         background: red;         position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */         top: 8px; bottom: 8px;    border-radius: 9px; }     QSlider::handle:horizontal {         width:15px;        border:1px;  background: #008792;         margin: -4px 0; /* expand outside the groove */    border-radius: 15px; }     QSlider::add-page:horizontal {         background: darkgray;     }     QSlider::sub-page:horizontal {         background:lightblue ;     }</string>
   </property>
   <property name="minimum">
    <number>-13</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>-4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>210</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LS</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderRS">
   <property name="geometry">
    <rect>
     <x>833</x>
     <y>240</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {         background: red;         position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */         top: 8px; bottom: 8px;    border-radius: 9px; }     QSlider::handle:horizontal {         width:15px;        border:1px;  background: #008792;         margin: -4px 0; /* expand outside the groove */    border-radius: 15px; }     QSlider::add-page:horizontal {         background: darkgray;     }     QSlider::sub-page:horizontal {         background:lightblue ;     }</string>
   </property>
   <property name="minimum">
    <number>-13</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>-4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>240</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>RS</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderRB">
   <property name="geometry">
    <rect>
     <x>833</x>
     <y>320</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {         background: red;         position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */         top: 8px; bottom: 8px;    border-radius: 9px; }     QSlider::handle:horizontal {         width:15px;        border:1px;  background: #008792;         margin: -4px 0; /* expand outside the groove */    border-radius: 15px; }     QSlider::add-page:horizontal {         background: darkgray;     }     QSlider::sub-page:horizontal {         background:lightblue ;     }</string>
   </property>
   <property name="minimum">
    <number>-13</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>-4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>290</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>LB</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderLB">
   <property name="geometry">
    <rect>
     <x>833</x>
     <y>290</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {         background: red;         position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */         top: 8px; bottom: 8px;    border-radius: 9px; }     QSlider::handle:horizontal {         width:15px;        border:1px;  background: #008792;         margin: -4px 0; /* expand outside the groove */    border-radius: 15px; }     QSlider::add-page:horizontal {         background: darkgray;     }     QSlider::sub-page:horizontal {         background:lightblue ;     }</string>
   </property>
   <property name="minimum">
    <number>-13</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>-4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>320</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>RB</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderW">
   <property name="geometry">
    <rect>
     <x>833</x>
     <y>400</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {         background: red;         position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */         top: 8px; bottom: 8px;    border-radius: 9px; }     QSlider::handle:horizontal {         width:15px;        border:1px;  background: #008792;         margin: -4px 0; /* expand outside the groove */    border-radius: 15px; }     QSlider::add-page:horizontal {         background: darkgray;     }     QSlider::sub-page:horizontal {         background:lightblue ;     }</string>
   </property>
   <property name="minimum">
    <number>-13</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>-4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>370</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSliderCenter">
   <property name="geometry">
    <rect>
     <x>833</x>
     <y>370</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {         background: red;         position: absolute; /* absolutely position 4px from the left and right of the widget. setting margins on the widget should work too... */         top: 8px; bottom: 8px;    border-radius: 9px; }     QSlider::handle:horizontal {         width:15px;        border:1px;  background: #008792;         margin: -4px 0; /* expand outside the groove */    border-radius: 15px; }     QSlider::add-page:horizontal {         background: darkgray;     }     QSlider::sub-page:horizontal {         background:lightblue ;     }</string>
   </property>
   <property name="minimum">
    <number>-13</number>
   </property>
   <property name="maximum">
    <number>4</number>
   </property>
   <property name="pageStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>-4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksAbove</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>400</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_L">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0 db</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_R">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>160</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0 db</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LS">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>210</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0 db</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RS">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>240</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0 db</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_LB">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>290</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0 db</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RB">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>320</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0 db</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_C">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>370</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0 db</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_W">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>400</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>0 db</string>
   </property>
  </widget>
  <widget class="QPushButton" name="counterClockwiseSurroundButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{border:0px; color:white;background:transparent;}
QPushButton:hover{background-color:red;}
QPushButton:checked{background-color:red;}</string>
   </property>
   <property name="text">
    <string>逆时针环绕</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clockwiseSurroundButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{border:0px; color:white;background:transparent;}
QPushButton:hover{background-color:red;}
QPushButton:checked{background-color:red;}</string>
   </property>
   <property name="text">
    <string>顺时针环绕</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="resetSurroundButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{border:0px; color:white;background:transparent;}
QPushButton:hover{background-color:red;}
QPushButton:checked{background-color:red;}</string>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="minusButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>90</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="increaseButton">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>90</y>
     <width>30</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="speakerPicture">
   <property name="geometry">
    <rect>
     <x>922</x>
     <y>90</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="dragAdjustVolumeButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{border:0px; color:white;background:transparent;}
QPushButton:hover{background-color:red;}
QPushButton:checked{background-color:red;}</string>
   </property>
   <property name="text">
    <string>拖拽模式</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
